client/src/pages/Login.js

```diff
@@ -35,9 +37,22 @@ const Title = styled.h1`
   justify-content: center;
 `;
 
+const SubmitButton = styled(CustomButton)`
+  color: #fff;
```

Comment:
Currently if you look at how `CustomButton` from `SubmitButton.js` is implemented, it accepts props that changes the theme of the button.

So instead of adding `color` and `background-color` here,  the relevant prop to use would be `primary`.